.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
-.env
\ No newline at end of file
+.env
+yarn.lock
```

Comment:
Please, do not ignore the lock files, they are here for a reason :)

Reply:
Good catch!
https://github.com/yarnpkg/yarn/issues/1583